Prog/Cvd19_affctd_ind_pop_tables.sas

```diff
@@ -0,0 +1,678 @@
+  weight perwt;
+  var pct_inc_less_cvd19_affctd_sum;
+run;
+
+
+** Worker and household tables **;
+
+options nodate nonumber;
+options orientation=portrait;
+
+%fdate()
+
+ods rtf file="&_dcdata_default_path\Covid19\Prog\Cvd19_affctd_ind_pop_tables.rtf" style=Styles.Rtf_arial_9pt;
+ods listing close;
+
+footnote1 height=9pt "Prepared by Urban-Greater DC (greaterdc.urban.org), &fdate..";
+footnote2 height=9pt j=r '{Page}\~{\field{\*\fldinst{\pard\b\i0\chcbpat8\qc\f1\fs19\cf1{PAGE }\cf0\chcbpat0}}}';
+footnote3 ' ';
+footnote4 '\b DRAFT - NOT FOR CITATION OR RELEASE';
+
+title2 ' ';
+
+
+**** Worker tables ****;
+
+title3 'Workers and earnings by COVID-19 affected industry status, MWCOG region';
+
+proc tabulate data=Covid19.cvd19_affctd_ind_pop format=comma16.0 noseps missing;
+  where mwcog_region = 1 and &CVD19_BOT_1PCT_EARNINGS < incearn < &CVD19_TOP_1PCT_EARNINGS;
+  weight perwt;
+  class cvd19_affctd_ind /order=data preloadfmt;
+  var total inctot incearn cvd19_affctd_incearn fulltime;
+  table 
+
+    /** Rows **/
+    all='Total' cvd19_affctd_ind=' ',
+
+    /** Columns **/
+    n='N (unwtd)' * total=' ' * f=comma8.0
+    sum='Workers' * total=' ' * f=comma10.0
+    
+    mean='Full time workers' * fulltime=' ' * f=percent10.0
+
+    sum='Annual income ($ 2018), 2014-18' * 
+    ( inctot='Total income' 
+      incearn='Total earnings'
+      cvd19_affctd_incearn='Earnings from COVID-19 affected industries' )
+
+    pctsum<inctot>='COVID-19 affected earnings as pct. total income' * 
+    cvd19_affctd_incearn=' ' *
+    f=comma12.1
+
+    pctsum<incearn>='COVID-19 affected earnings as pct. total earnings' * 
+    cvd19_affctd_incearn=' ' *
+    f=comma12.1
+  ;
+  table 
+
+    /** Rows **/
+    all='Total' cvd19_affctd_ind=' ',
+
+    /** Columns **/
+    n='N (unwtd)' * total=' ' * f=comma8.0
+    sum='Workers' * total=' ' * f=comma10.0
+    
+    mean='Full time workers' * fulltime=' ' * f=percent10.0
+
+    mean='Annual income per capita ($ 2018), 2014-18' * 
+    ( inctot='Total income' 
+      incearn='Total earnings'
+      cvd19_affctd_incearn='Earnings from COVID-19 affected industries' )
+
+    pctsum<inctot>='COVID-19 affected earnings as pct. total income' * 
+    cvd19_affctd_incearn=' ' *
+    f=comma12.1
+
+    pctsum<incearn>='COVID-19 affected earnings as pct. total earnings' * 
+    cvd19_affctd_incearn=' ' *
+    f=comma12.1
+  ;
+  format cvd19_affctd_ind cvd19_affctd_ind.;
+run;
+
+
+title3 'Workers and earnings in COVID-19 affected industries by industry type, MWCOG region';
+
+proc tabulate data=Covid19.cvd19_affctd_ind_pop format=comma16.0 noseps missing;
+  where mwcog_region = 1 and cvd19_affctd_ind = 1 and &CVD19_BOT_1PCT_EARNINGS < incearn < &CVD19_TOP_1PCT_EARNINGS;
+  weight perwt;
+  class ind /order=data preloadfmt;
+  var total inctot incearn cvd19_affctd_incearn fulltime;
+  table 
+
+    /** Rows **/
+    all='Total' ind='\line \i By industry type',
```

Comment:
The `\line \i` syntax is causing some weird formatting on the RTF that I'm getting. Now sure if it's specific to me or not.

Reply:
@rpitingolo Yes, you have to use the Styles.Rtf_arial_9pt style to get those codes to resolve properly and you have to batch submit General\Prog\Style_rtf_arial_9pt.sas to create that style. I should have put that in the program header so I'll add. that.